staging/src/k8s.io/apiserver/pkg/endpoints/filters/audit_test.go

```diff
@@ -852,3 +851,25 @@ func withTestContext(req *http.Request, user user.Info, audit *auditinternal.Eve
 	}
 	return req.WithContext(ctx)
 }
+
+func TestResponseWriterDecorator(t *testing.T) {
+	decorator := &auditResponseWriter{
+		ResponseWriter: &responsewriter.FakeResponseWriter{},
+	}
+	var w http.ResponseWriter = decorator
+
+	if inner := w.(responsewriter.Unwrapper).Unwrap(); inner != decorator.ResponseWriter {
+		t.Errorf("Expected the decorator to return the inner http.ResponseWriter object")
+	}
```

Comment:
This check would make sense in the previous test func; if it were there then we would not also need this test func.